patch/0026-platform-x86-mlx-platform-Remove-PSU-EEPROM-configur.patch

```diff
@@ -0,0 +1,48 @@
+From 6a826ae7e734054d610f0d3a9daad88339ee35e7 Mon Sep 17 00:00:00 2001
+From: Oleksandr Shamray <oleksandrs@nvidia.com>
+Date: Wed, 21 Oct 2020 15:10:48 +0300
+Subject: [PATCH 2/2] platform/x86: mlx-platform: Remove PSU EEPROM
+ configuration
+
+Remove PSU EEPROM configuration for systems class equipped with
+Mellanox chip Spectrum and Mellanox Broadwell Comex. Till now all the systems from this class
+used few types of power units, all equipped with EEPROM device with
+address space two bytes. Thus, all these devices have been handled by
+EEPROM driver "24c02".
+There is a new requirement is to support power unit replacement by "off
+the shelf" device, matching electrical required parameters. Such device
+could be equpped with different EEPROM type, which could be one byte
+address space adressing or even could be not equipped with EEPROM.
+In such case "24c02" will not work.
+
+Fixes: ef08e14a3 ("platform/x86: mlx-platform: Add support for new msn274x system type")
+Signed-ff-by: Oleksandr Shamray <oleksandrs@nvidia.com>
```

Comment:
Signed-off-by line typo.

Reply:
patch has been re-generated with the upstream commit.